### PR TITLE
Removed ReleaseNotes

### PR DESCRIPTION
**Pull request by Cronan**
Not required

### CODE REVIEW

File: src/NArctic/NArctic.nuspec [hunk 1]
```diff
@@ -10,8 +10,7 @@
     <projectUrl>https://github.com/manahl/NArctic</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>A port of the python Arctic library to DotNet</description>
-    <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
     <copyright>Copyright Man Group plc 2018</copyright>
-    <tags>Arctic C# MongoDb DotNet</tags>
+    <tags>Arctic C# MongoDb DotNet</tags
```
Comment (DrNickClarke) [inline src/NArctic/NArctic.nuspec]: is there a close tag missing here?

**Review by DrNickClarke — CHANGES_REQUESTED** [review]

File: src/NArctic/NArctic.nuspec (hunk 1 above)
Comment (Cronan) [reply src/NArctic/NArctic.nuspec]: oops

**Review by Cronan — COMMENTED** [review]
Closing tag issue fixed

**Review by DrNickClarke — APPROVED** [review]
Looks good now

**Pull request merged**